plotly/tools.py

```diff
@@ -2081,4 +2104,3 @@ def sum_streamlines(self):
         streamline_x = sum(self.st_x, [])
         streamline_y = sum(self.st_y, [])
         return streamline_x, streamline_y
-
```

Comment:
:cow2: you should be able to set your editor so that there's always a newline added at the end of a file on save (if it's not already there)